barretenberg/cpp/src/barretenberg/vm/avm/tests/arithmetic.test.cpp

```diff
@@ -323,12 +323,30 @@ class AvmArithmeticTestsU64 : public AvmArithmeticTests {};
 class AvmArithmeticTestsU128 : public AvmArithmeticTests {};
 class AvmArithmeticTestsDiv : public AvmArithmeticTests, public testing::WithParamInterface<ThreeOpParamRow> {};
 
-class AvmArithmeticNegativeTestsFF : public AvmArithmeticTests {};
-class AvmArithmeticNegativeTestsU8 : public AvmArithmeticTests {};
-class AvmArithmeticNegativeTestsU16 : public AvmArithmeticTests {};
-class AvmArithmeticNegativeTestsU32 : public AvmArithmeticTests {};
-class AvmArithmeticNegativeTestsU64 : public AvmArithmeticTests {};
-class AvmArithmeticNegativeTestsU128 : public AvmArithmeticTests {};
+class AvmArithmeticNegativeTestsFF : public AvmArithmeticTests {
+  protected:
+    void SetUp() override { GTEST_SKIP(); }
+};
+class AvmArithmeticNegativeTestsU8 : public AvmArithmeticTests {
```

Comment:
@IlyasRidhuan Can we try to keep at least a very basic negative test per type? Like just mutating the output to a wrong value (only if this can be achieved in a prompt way).

Reply:
Added an issue